src/wallet/wallet2.cpp

```diff
@@ -772,6 +790,60 @@ void wallet2::check_pending_txes()
   }
 }
 //----------------------------------------------------------------------------------------------------
+void wallet2::fast_refresh(uint64_t start_height, uint64_t &blocks_start_height, std::list<crypto::hash> &short_chain_history)
+{
+  std::list<crypto::hash> hashes;
+  size_t current_index = m_blockchain.size();
+  while(current_index < m_refresh_from_block_height)
+  {
+    pull_hashes(start_height, blocks_start_height, short_chain_history, hashes);
```

Comment:
That's intentional. Perhaps I should just get rid of the start_height parameter. (pull_hashes was just a copy/paste of pull_blocks.) If you actually increment the start_height parameter, everything breaks, same as running refresh with a non-zero start_height.